brainiak/searchlight/searchlight.py

```diff
@@ -306,6 +303,16 @@ def distribute(self, subjects, mask):
         mask: 3D array with "True" entries at active vertices
 
         """
+        if mask.ndim != 3:
+            raise ValueError('mask should be a 3D array')
+
+        for (idx, subj) in enumerate(subjects):
+            try:
+                if subj.ndim != 4:
+                    raise ValueError('subjects[{}] must be 4D'.format(idx))
+            except AttributeError:
```

Comment:
Given that we are expecting `None` here, we should explicitly check for it, instead of catching the `AttributeError`.